clustermesh-apiserver/vmmanager.go

```diff
@@ -274,8 +274,16 @@ const (
 
 // UpdateCiliumNodeResource updates the CiliumNode resource representing the
 // local node
-func (m *VMManager) UpdateCiliumNodeResource(n *nodeTypes.RegisterNode) {
+func (m *VMManager) UpdateCiliumNodeResource(n *nodeTypes.RegisterNode, cew *ciliumv2.CiliumExternalWorkload) {
 	nr := n.ToCiliumNode()
+	nr.OwnerReferences = []metav1.OwnerReference{
+		{
+			APIVersion: ciliumv2.SchemeGroupVersion.String(),
+			Kind:       "CiliumExternalWorkload",
```

Comment:
Would `"CiliumExternalWorkload"` be better represented using the `ciliumv2.CEWKindDefinition` constant instead?

Reply:
you are completely right, let me make an update. Thanks for your help raising issue and proposing approach again.